mobify/progressive-web-scaffold
# Update icons and icon sprite

Update icons to match how icons work in the most current UI Kit

## Changes
- Add new icons
- Replace old icons with new icons, including update names
- SVG icons no longer have built in padding, that must be added via CSS
- Remove SVG files that are no longer necessary

## Todo
- [x] Add SVGO or some kind of automated SVG optimization tool

## How to test-drive this PR
- Run `npm install` (new SVGO module added)
- Preview Merlin's Potions
- Try out some icons and ensure that they work as intended:
    - alert, caret-down, caret-left, caret-right, caret-up, cart-full, cart, caution, check, chevron-down, chevron-left, chevron-right, chevron-up, chevron, close, done, edit, help, info, like, lock, logo, map, menu, minus, more, payment-full, payment, plus, search, shipping-full, shipping, star, store, trash, user, zoom
- Run `npm run build-sprites`. SVG sprite should be generated (although it does already exist). Confirm it builds correctly!
